db/migrate/20160627152327_add_pii_table.rb

```diff
@@ -0,0 +1,39 @@
+class AddPiiTable < ActiveRecord::Migration
+  # rubocop:disable AbcSize, MethodLength
+  def up
+    create_table :piis do |tbl|
+      tbl.integer :user_id, null: false
+      tbl.boolean :active, null: false, default: false
+      tbl.boolean :verified, null: false, default: false
+      tbl.timestamps null: false
+
+      tbl.string :first_name
+      tbl.string :middle_name
```

Comment:
I think middle name might not be in our core attribute bundle either.  We really should sync with the rest of the team about the core attribute set that was settled on.